### ISSUE TITLE
How to implement only update c when minimizing KL divergence?

### ISSUE THREAD

**Issue by Thinking-more** [yuleiniu/cfvqa#15]
Hello Yulei Niu. Thank you very much for your inspiring work, it has given me great inspiration. I have a question that I hope you can answer: In your paper, under equation 17, there is a sentence "Only c is updated when minimizing L_kl", but in the code, I don't see how it is implemented, it seems that L_kl is added to the whole loss.

**Comment by yuleiniu** [comment]
We detached the gradients of the logits output:

https://github.com/yuleiniu/cfvqa/blob/736795bef50db9c76818f9a08202c7f325489afd/cfvqa/models/networks/cfvqa.py#L85

**Comment by Thinking-more** [comment]
I'm still a little confused. Could you describe it more clearly? Why this operation can  implement only update c when minimizing KL divergence.

**Comment by Thinking-more** [comment]
I see, thank you for your kind reply
